test_terra_accounts/settings.py

```diff
@@ -146,3 +146,7 @@
     'JWT_PAYLOAD_HANDLER': 'terra_accounts.jwt_payload.terra_payload_handler',
     'JWT_EXPIRATION_DELTA': timedelta(seconds=9999)
 }
+
+TERRA_APPLIANCE_SETTINGS = {
+    'disabled_modules': [],
+}
```

Comment:
It is better to have a new line at the end of a file (but not a big deal).

Reply:
I add a newline at the end of the file